### ISSUE TITLE
Liveness analysis false-positive "parameter is never used"

### ISSUE THREAD

**Issue by vlad20012** [intellij-rust/intellij-rust#5420]
## Environment

* **IntelliJ Rust plugin version:** 0.2.123.3108-201-nightly
* **Rust toolchain version:** 1.42.0 (b8cedc004 2020-03-09) x86_64-unknown-linux-gnu
* **IDE name and version:** IntelliJ IDEA 2020.1.1 Ultimate Edition (IU-201.7223.91)
* **Operating system:** Linux 5.4.0-29-generic

## Steps to reproduce
```rust
struct Foo { field: i32 }
fn foo(Foo { field }: Foo) {
    let _ = field;
}
```    

![image](https://user-images.githubusercontent.com/3221931/82655180-7729a980-9c2a-11ea-95b6-33cd6e8d4475.png)

**Comment by Kobzol** [comment]
I think that this has been already reported in https://github.com/intellij-rust/intellij-rust/issues/5109.

**Comment by Undin** [comment]
@Kobzol sorry, I forgot to provide you permissions to manage issues. Too many places with different permissions :sweat_smile:
Could you check it now please?
We usually marked such issues as [duplicate](https://help.github.com/en/github/managing-your-work-on-github/about-duplicate-issues-and-pull-requests), add `duplicate` label and close

**Comment by Kobzol** [comment]
Duplicate of https://github.com/intellij-rust/intellij-rust/issues/5109
